theme/ckeditor5-ui/components/tooltip/tooltip.css

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Why did you decide to make a folder for `tooltip.css` instead of placing it directly into `/components`?

Reply:
Check out the structure of theme in `ckeditor5-ui`. AFAIR the tooltip has a mixin and thus it has been defined as a directory (multiple files). The theme must follow such pattern if it is to be loaded properly. 

TBH, now that I think about it, I think each component in ckeditor5-ui should have its own directory because if one day it gets a mixin or whatever, we wouldn't need to fix a structure of N–themes to reflect ckeditor5-ui and make them work with the theme importer. If you know what I mean :P

Reply:
Ok, I see this and it's clear to me 👍